README.md

```diff
@@ -15,10 +15,10 @@ Another good example is to think at Redis as a more complex version of memcached
 
 If you want to know more, this is a list of selected starting points:
 
-* Introduction to Redis data types. http://redis.io/topics/data-types-intro
-* Try Redis directly inside your browser. http://try.redis.io
-* The full list of Redis commands. http://redis.io/commands
-* There is much more inside the Redis official documentation. http://redis.io/documentation
+* [Introduction to Redis data types][4].
+* [Try Redis directly inside your browser][5]. http://try.redis.io
+* [The full list of Redis commands][6]. http://redis.io/commands
+* [There is much more inside the Redis official documentation][7]. http://redis.io/documentation
```

Comment:
Either the urls at the end or the full sentence linked, not a mix of both and also not randomly removing the visible url from the first one